src/amo/components/Link.js

```diff
@@ -1,5 +1,4 @@
-import path from 'path';
-
+import join from 'join-url';
```

Comment:
Works for me actually, will do. This is just copied from their example, I didn't give it any thought but you're right.